webui-src/app/theme.css

```diff
@@ -327,17 +332,18 @@ a.tab-menu-item {
 }
 .tab-menu-item:hover {
   background-color: #999;
+  width: -webkit-fill-available;
```

Comment:
So this will not work on firefox, a better solution would be to set the width along with padding & margin in `.tab-menu-item` class itself:
```diff
@@ -318,7 +316,9 @@ nav.tab-menu {
   margin-right: 0rem;
 }
 a.tab-menu-item {
-  padding: 1em;
+  padding: 1em 0;
+  margin: 0 1em;
+  width: 100%;
   text-align: center;
   text-decoration: none;
   color: #ccc;
@@ -332,18 +332,18 @@ a.tab-menu-item {
 }
 .tab-menu-item:hover {
   background-color: #999;
-  width: -webkit-fill-available;
 }
 
 .tab-menu-item.selected-tab-item {
   background-color: #3ba4d7;
   color: white;
-  width: -webkit-fill-available;
 }
```